src/tetris/brick/block.ts

```diff
@@ -17,18 +17,30 @@ export default class Block {
 	//endregion
 
 	//region public methods
+	public update(): void {
+	}
+	
+	public preDraw(brickPosition: Vector2, fieldOffset: Vector2) {
+		const fieldPosition = brickPosition.clone().add(this._position);
+		const pixelPosition = fieldPosition.clone().scale(this._sprite.frame.cutHeight).add(new Vector2(this._sprite.frame.cutWidth, this._sprite.frame.cutHeight).scale(0.5)).add(fieldOffset);
```

Comment:
Please add linke breaks between different functions calls in order to increase readability.